crates/api_crud/src/user/my_user.rs

```diff
@@ -33,6 +34,9 @@ pub async fn get_my_user(
     ))
     .with_lemmy_type(LemmyErrorType::SystemErrLogin)?;
 
+  if context.settings().disable_donation_dialog.unwrap_or(false) {
+    local_user_view.local_user.last_donation_notification = Utc::now();
+  }
```

Comment:
Probably unnecessary. Front ends need to have both the site and `MyUser` responses anyway, so they can check the `disable_donation_dialog` on `local_site` `before showing a popup.

Reply:
Its a config value so its not part of the api.